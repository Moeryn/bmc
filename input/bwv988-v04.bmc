3#c(
  $`.x!jbh $el!i$e2`x_`!ebga $8."c2`_`_jb$`!ha$dl $hgx2`!}{ix m2`m m2`x!dlf
x!ild"c2`_ib!5"c $y~jx2`!ehx
  _`_8.a 7.kc, 6.2`x_jah x_e,g2`!5x >w.2`x_hj _4.,ck2`_fihbca"c _5.2`_h%&g
x>j,ea2`_hex
  $`.x$flh2`m 3$d$92`'m $ea8"c2`x3$df $hgfk"c2`!9h"c $f)e,)1dk"c0b d:wk*b'0l
i,3)$e#l3d#$c #,.3$dex2`!{&gx2; #;.=2k.
  _`>h_8ac,"c2`'m _h7"c2`x!ebg _g,fh2`_jb!fx 3_d_i,3d2`'x3!df _7.k0l"c h>8"c2`
_ih$`_j _`>h)g,-ib* #,>e._z,&%2`_5x2; #;>e._~,}3y2`_5x2k.
  $`.2=x$ge2`'m $hw"c2`x$ej $j~,{k%&k2`_`!h$`!id 3$zl&7"c2`!7x $gf,h2`x!jh
3$dl)3e)f"c) f3z3y}k~2`!7. !+!{%fx2`!6x
  _`.2=!51dl2`'m _wh2`x!eh !dl_gki2`!f4a"c _j3_ej2`!d}b~a{% _f,hf2`_{a~bw"c
_i|g0f* il0bja_a"c>j _6l*'0e)
  $`.x$d0f0, !il0je !hl$4,"c d0e)d0"c }y5"c2`!ihj $zdedji2`!6. $e!hkg,"c2`
_`_ib$`!eadb"c #,!ghx2`!y~jx2; #;.=2k
  _`_4."c2`_f~{%{a _dkclw"c2`$`!5. >jid,2`$`!d_`_gbi >gleg,2`_5."c >hkjh2`_5h"c
_4.lck"c2`_hibd _d#j)aeb* #,>h._~,}y2`_8x2; #;>8x2`_8'x2k

