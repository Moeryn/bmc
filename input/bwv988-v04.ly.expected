% Automatically generated by BMC, the braille music compiler
\version "2.14.2"
\score {
  <<
    \new PianoStaff <<
      \new Staff {
        \clef "treble"
        \key g \major
        \time 3/8
        r8 b'8-2 g'8 | % 1
        << {d''8-3 a'8 d''8}\\{r8  d'8-2 fis'8-1} >> | % 2
        << {g''4.~}\\{ b8-2  g'8-1 c''8-3} >> | % 3
        << {g''8 fis''8 r8}\\{b'16 g'16 a'8 r8} >> | % 4
        << {r8*3}\\{r8*3} >> | % 5
        << {r8*3}\\{r8 c'8-3 e'8} >> | % 6
        << {r8 a'8-3 c''8~}\\{a8-2 d'4~} >> | % 7
        << {c''16 a'16 b'8 r8}\\{d'8 g'8 r8} >> | % 8
        << {r8 e''8-3 g''8}\\{r8*3} >> | % 9
        << {cis''8 a''4}\\{r8*3} >> | % 10
        << {d''8-1 g''4~}\\{r8 cis''8 e''8} >> | % 11
        << {g''8 fis''8 e''8~-5}\\{a'4 g'8~} >> | % 12
        <e'' g'>8 <d''-4 fis'>8 <c''~-5 a'-2>8 | % 13
        <c'' d'>8 <b'-5 e'-2 c''''-3>4 | % 14
        <a'-4 cis'>8 <d'' a'-3>8 <cis'' g'>8 | % 15
        << {cis''8 d''8 r8}\\{g'16 e'16 fis'8 r8 } >> \bar ":|" % 16
        << {cis''8 d''8 r8}\\{g'16 e'16 fis'8 r8 } >> \bar "|:" % 17
        << { r8 fis''8 d''8}\\{r8*3} >> | % 18
        << {g''8 b''4~}\\{r8 d''8 b'8} >> | % 19
        << {b''8 a''16-4 g''16-5 fis''16 e''16-5}\\{ g'8  a'8 c''8} >> | % 20
        << {dis''16-3 e''16 fis''4~}\\{fis'4 r8} >> | % 21
        << {fis''8 e''8-4 g''8}\\{r8 b'8 g'8} >> | % 22
        <cis''-3 e'>8 <dis'' fis'>8 <e''~ g'>8 | % 23
        << {e''8 dis''16 cis''16 b'16-5 a'16}\\{fis'4.} >> | % 24
        << {g'16 fis'16 e'8 r8}\\{e'4 r8} >> | % 25
        r8 <c'' a'>8 <e'' c''-4>8 | % 26
        <a'-3 fis'>8 b'8 d''8 | % 27
        g'8-3 c''4~-4 | % 28
        <c'' a'>8 <d'' fis'>8 <c'' a'~>8 | % 29
        << {b'16 c''16 d''4~}\\{a'8 g'8 b'8} >> | % 30
        << {d''16 c''16 d''16 c''16 b'16 a'16}\\{e'4.} >> | % 31
        << {d''8 g'8-5 fis'8~-4}\\{ a8-2  d'8-1 c'8~-2} >> | % 32
        << {fis'8 g'8 r8}\\{c'16 a16 b8 r8 } >> \bar ":|" % 33
        << {fis'8 g'8 r8}\\{c'16 a16 b8 r8} >> | % 34
      }
      \new Staff {
        \clef "bass"
        \key g \major
        \time 3/8
        g4.-1 | % 1
        fis4.-"5-4" | % 2
        << {e4.}\\{r8 b8-1 g8} >> | % 3
        << {r8 d8-4 fis8}\\{d'4 r8} >> | % 4
        << {b,4.}\\{r8 g8 b8} >> | % 5
        << {c4.-"4-5"}\\{e8 a8 g8~-"2-1"} >> | % 6
        << {d4.}\\{g8 fis16 e16 fis8} >> | % 7
        << {r8 b,8-4 d8-1}\\{g8 d8 r8} >> | % 8
        << {g,8 g4~-"1-4"}\\{r8*3} >> | % 9
        << {g8 fis4~}\\{r8 d'8-2 fis'8} >> | % 10
        << {fis8-4 e8 g8}\\{b8-2 e'8 r8} >> | % 11
        << {cis8 a8-4 cis'8}\\{r8 cis'8 e'8} >> | % 12
        <fis-5 a~-3>4. | % 13
        << {g8 g,4~}\\{a8 g8  b8} >> | % 14
         <g, e>8 <fis,-4 fis>8 <a,-2 e>8 | % 15
        << {d,8. d16-4 e16 fis16}\\{d4 r8 } >> \bar ":|" % 16
        << {d,8. a16-4 b16 cis'16}\\{d4 r8 } >> \bar "|:" % 17
        << { d'4 c'8-3}\\{r8*3} >> | % 18
        << {b4 g8}\\{r8 d'8 g'8} >> | % 19
        << {c'8-3 fis8-5 a8}\\{e'8 c'4~-1} >> | % 20
        << {b8 dis8 b,8}\\{c'8 b16-2 a16-1 g16 fis16} >> | % 21
        << {e8-4 g8 e8}\\{g16-1 a16-2 b4~} >> | % 22
        <a b>8 <fis a>8 <e b>8 | % 23
        <a-3 c'-2>8 b8-1 b,8 | % 24
        <e-3 b d''''>4 <d b>8 | % 25
        << {c4.~}\\{e8 a16 g16 fis16 g16-1} >> | % 26
        << {c8-"5-3" b,4~}\\{ d'4.} >> | % 27
        << {b,8 a,8 c8-4}\\{ c'8  fis8-2 a8} >> | % 28
        << {fis,8-3 d,8 fis,8-4}\\{d4.~} >> | % 29
        << {g,8-5 b,8 g,8}\\{d4 g8~} >> | % 30
        << {c4.~-"3-5"}\\{g8 a8-2 c'8} >> | % 31
        <c fis>8 <b, g-1>8 <d-2 a>8 | % 32
        << {g,8. a16-4 b16 c'16}\\{g4 r8 } >> \bar ":|" % 33
        << {g,4 r8}\\{g4 r8} >> | % 34
      }
    >>
  >>

  \layout { }
  \midi { }
}
